remo/sdk.py

```diff
@@ -762,3 +763,61 @@ def view_annotation_stats(self, annotation_set_id: int):
 
     def _view(self, url, *args, **kwargs):
         return self.viewer.browse(self.api.url(url, *args, **kwargs))
+    
+    def generate_annotations_from_folders(self, path_to_data_folder: str, output_filename="image_classification", generate_encoding=False):
```

Comment:
```suggestion
    def generate_annotations_from_folders(self, path_to_data_folder: str, output_filename="img_class_annotations.csv", generate_encoding=False):
```

Reply:
This no longer applies with latest changes.